### PR TITLE
Add ability to set document keywords.

### PR DESCRIPTION
**Pull request by qrnch-jan**
_Heavily_ modeled after `author`, but uses plural `keywords` because my gut feeling is that it's more common to have multiple keywords (for those who bother to use the field).

(Also ignore `.*.swp` files in `.gitignore`)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # General
 .vscode
 .idea
+.*.swp
```
Comment (LuxxxLucy) [inline .gitignore:4]: Other than the gitignore (just revert it), all seems fine to me.

Comment (qrnch-jan) [reply .gitignore]: I rarely use git, so I'm not entirely sure I did the right thing.  I `git revert`'ed the commit hash and pushed the changes.  If some other action needs to be taken, let me know.

File: crates/typst/src/export/pdf/mod.rs [hunk 2]
```diff
@@ -132,6 +132,14 @@ fn write_catalog(ctx: &mut PdfContext) {
         info.author(TextStr(&authors.join(", ")));
         xmp.creator(authors.iter().map(|s| s.as_str()));
     }
+
+    let keywords = &ctx.document.keywords;
+    if !keywords.is_empty() {
+        info.keywords(TextStr(&keywords.join(", ")));
+        let kw = keywords.join(", ");
+        xmp.pdf_keywords(&kw);
```
Comment (laurmaedje) [inline crates/typst/src/export/pdf/mod.rs:140]:
```suggestion
        let joined = keywords.join(", ");
        info.keywords(TextStr(&joined));
        xmp.pdf_keywords(&joined);
```

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**